Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,17 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const hourTasks = tasks.map(elem => elem.duration / 60);
+  const moreThan2Hour = hourTasks.filter(elem => (elem >= 2 ? elem : false));
```

Comment:
You already are filtering things so it will only return an element if the element is equal or greater than 2 (`elem >= 2`). There is no need the return false because the filter function will just not return that element. 

`hourTasks.filter(elem => elem >= 2);`